### DISCUSSION TITLE
Docker container, persist data.

### DISCUSSION THREAD

**Discussion by Unrepentant-Atheist** [Lissy93/dashy#309] [Help and Support] [answered]
How do I persist the data? All these edits I made, I can download them into a dashy-conf.yml, but that's it. There's no folder, no volume, nothing.

When I do
```
     volumes:
       - /path/to/my-config.yml:/app/public/conf.yml
```

I get: https://www.toptal.com/developers/hastebin/aqejudamuw.txt , and that's just the default dashboard exported, nothing edited.

**Comment by lissy93** [comment, marked as answer]
That looks like a YAML syntax error, probably an extra / missing indentation somewhere. If you paste your config into [this validator](https://jsonformatter.org/yaml-validator), it should show you what's up.

Were you using the interactive editor when you made the export?

**Reply by Unrepentant-Atheist** [reply]
>Were you using the interactive editor when you made the export?

Yes, but yes, the validator helped. Was indeed a indentation.
